app/templates/tools/includes/list.html

```diff
@@ -46,14 +46,18 @@ <h2 class="heading-medium">Your tools</h2>
           Not deployed
         {% endif %}
       </td>
+        {% for availableTool in availableTools %}
+      <tr>
       <td class="align-right no-wrap">
         {% if not rstudio_is_deploying %}
```

Comment:
Maybe this `if` should also check if tool currently rendered is RStudio, e.g.:

```python
{% if availableTool.tool == 'rstudio' and not rstudio_is_deploying %}
...
```

(or something like that)